### PR TITLE
#166153214 Fix failing circleci tests 

### PR DESCRIPTION
**Pull request by kizzanaome**
#### What Does This PR Do?
- Fix Failing Circle ci Tests

### CODE REVIEW

File: server/a_socials/settings/prod.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     'default': dj_database_url.config(default=dotenv.get('DATABASE_URL'))
 }
 
-db_from_env = dj_database_url.config(conn_max_age=500)
+db_from_env = dj_database_url.config(conn_max_age=None)
```
Comment (andela-sjames) [inline server/a_socials/settings/prod.py]: Prod settings are not used in the test if I recall correctly.

File: server/graphql_schemas/tests/attend/base.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def setUp(self):
         self.request = RequestFactory().get('/graphql')
         self.client = Client(schema)
 
-    def tearDown(self):
-        Category.objects.all().delete()
-        AndelaUserProfile.objects.all().delete()
-        Event.objects.all().delete()
+    # def tearDown(self):
```
Comment (andela-sjames) [inline server/graphql_schemas/tests/attend/base.py]: You can also skip this in the meantime using this: https://docs.djangoproject.com/en/2.2/topics/testing/tools/#skipping-tests

Comment (andela-sjames) [reply server/graphql_schemas/tests/attend/base.py]: https://docs.djangoproject.com/en/2.2/topics/testing/tools/#skipping-tests

File: server/a_socials/settings/prod.py (hunk 1 above)
Comment (okellogabrielinnocent) [reply server/a_socials/settings/prod.py]: Thanks at @andela-sjames the work on this PR is not complete and we're trying to figure out why the tests are failing. 
This is helpful feedback

File: server/graphql_schemas/tests/attend/base.py (hunk 2 above)
Comment (okellogabrielinnocent) [reply server/graphql_schemas/tests/attend/base.py]: Thanks, @andela-sjames for this has been helpfull

**Comment by kizzanaome** [comment]
Hi, @andela-sjames. There is another temporary solution apart from skipping the tests, we can preserve the django test database after running the test cases. by using the test --keepdb option. I added that option on the command that runs the tests on circleCi.And Circle ci able to pass.
image.

**Comment by brianbirir** [comment]
Thanks @kizzanaome. Looks good. Let's hope this works.

**Pull request merged**